Generate content of the knowledge base index page (#794)
Avaiga/taipy-doc: docs/knowledge_base/tips/skippable_tasks/index.md

---
title: Skippable tasks
category: tips
type: code
data-keywords: scenario task datanode job submission configuration

> short-description: Explore the 'skippable' task feature to improve your efficiency in pipeline orchestration.

Reviewer: ```suggestion
short-description: Improve the efficiency in pipeline orchestration with the 'skippable' task feature.
```
